rules_daml/daml.bzl

```diff
@@ -61,10 +60,6 @@ _daml_configure = rule(
         "target": attr.string(
             doc = "DAML-LF version to output.",
         ),
-        "ghc_options": attr.string_list(
```

Comment:
we set this twice once in `_daml_build` and once in `daml_configure` which broke passing `--log-level` since the flag is only accepted once (as all other flags).